atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -264,8 +264,10 @@ public void successfulRequestCausesHostToBeRemovedFromBlacklist() {
 
     @Test
     public void resilientToRollingRestarts() {
-        CassandraClientPool cassandraClientPool =
-                clientPoolWithServersInCurrentPool(ImmutableSet.of(CASS_SERVER_1, CASS_SERVER_2));
+        when(config.autoRefreshNodes()).thenReturn(false);
```

Comment:
nit: This was not necessarily needed, since it was already returning false. But adding it for clarity, since when investigating the behaviour wasn't obvious to me: [despite the default value for such config being True](https://github.com/palantir/atlasdb/blob/c40656a1424dce596e22813bf73603f4f938e353/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java#L290-L293), if you don't provide the return for a mocked a boolean method, it will return false.